src/database/migrations/20191225101900-create-end-of-call-state.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.createTable('End_of_call_states', {
```

Comment:
I understand that you were going for a translation of "stanje", but "state" has a very well-understood meaning in programming, so I think avoiding usage of the word "state" in our schema. Maybe something like `Call_resolution_type`?

Reply:
Sure